crates/stackable-certs/CHANGELOG.md

```diff
@@ -4,13 +4,25 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- BREAKING: `CertificateAuthority::generate_leaf_certificate` (and `generate_rsa_leaf_certificate` and `generate_ecdsa_leaf_certificate`)
+  now take an additional parameter `subject_alterative_dns_names`. The passed SANs are added to the generated certificate,
+  this is needed when the HTTPS server is accessible on multiple DNS names and/or IPs.
+  Pass an empty list (`[]`) to keep the existing behavior ([#1057]).
```

Comment:
note: This is a change, not an addition. As such, this should live in the "Changed" section.

suggestion: Slight reword.

```suggestion
- BREAKING: The functions `generate_leaf_certificate`, `generate_rsa_leaf_certificate` and
  `generate_ecdsa_leaf_certificate` of `CertificateAuthority` accept an additional parameter
  `subject_alterative_dns_names` ([#1057]).
  - The passed SANs are added to the generated certificate, this is needed when the HTTPS server is
    accessible on multiple DNS names and/or IPs.
  - Pass an empty list (`[]`) to keep the existing behavior.
```